Extension/src/githubAPI.ts

```diff
@@ -114,7 +114,15 @@ function getArrayOfBuilds(input: any): Build[] {
 export function vsixNameForPlatform(info: PlatformInformation): string {
     const vsixName: string | undefined = function(platformInfo): string | undefined {
         switch (platformInfo.platform) {
-            case 'win32': return 'cpptools-win32.vsix';
+            case 'win32':
+                switch (platformInfo.architecture) {
+                    case 'x86_64': return 'cpptool-win32.vsix'; // TODO: Change to cpptools-win64?
```

Comment:
It looks like this is misspelled

Reply:
This needs to be fixed or else insiders will be broken for x64.

Reply:
yikes, good catch.